portfolio/src/components/Info.tsx

```diff
@@ -1,131 +0,0 @@
-                                        )
-                                    })}
-                                </Col>
-                                <Col className="skill-list">
-                                    {this.state.misc.map((s, i) => {
-                                        return (
-                                            <li className="misc" key={i}><div className="level-style" style={this.getLevelStyles(s.level, "#74a09c")}>{s.name}</div></li>
-                                        )
-                                    })}
-                                </Col>
-                            </Row>
-                        </div>
-                        <div className="about-section info-more">
-                            <h3 className="sub-header">Contact</h3>
-                            <p>Let's connect!</p>
-                            <ul className="links-container">
-                                <li className="panel"><Button variant="outline-light" onClick={() => window.open("https://www.linkedin.com/in/jjennha", "_blank")}>LinkedIn</Button>{' '}</li>
-                                <li className="panel"><Button variant="outline-light" onClick={() => window.open("https://twitter.com/jjennha", "_blank")}>Twitter</Button>{' '}</li>
-                                <li className="panel"><Button variant="outline-light" onClick={() => window.open("https://github.com/jjennha", "_blank")}>GitHub</Button>{' '}</li>
-                                <li className="panel"><Button variant="outline-light" onClick={() => window.open("http://jennha.com/Resume1.pdf", "_blank")} title="Resume">Resume</Button>{' '}</li>
-                            </ul>
-                        </div>
-                    </Col>
-                    <Col xl={true} className="info-panel info-timeline">
-
-                        <div id="timeline">
-                            <div className="timeline-header">
-                                <h3 className="header">Updates</h3>
-                                <p>Some exciting professional and personal milestones on my timeline.</p>
-                            </div>
-                            <ul id="drop">
-                                {this.state.updates.map((u, i) => {
-                                    return (
-                                        <li className="" key={i}>
-                                            <div className="date">{u.date}</div>
-                                            <div className="event-desc">
-                                                <div className="title">{u.event}</div>
-                                                <div className="details" dangerouslySetInnerHTML={{ __html: u.details }} />
-                                            </div>
-                                        </li>
-                                    )
-                                })}
```

Comment:
I would split these into separate components each with their own state.

Reply:
For example, let's say the expanding on hover thing you wanted to happen on a state change instead of with CSS, you wouldnt be able to do that unless you did it in a new component

Reply:
Recommended to try that to learn react hooks